packages/lib/server/queries/teams/index.ts

```diff
@@ -87,6 +87,7 @@ export async function getTeamWithMembers(args: {
       name: true,
       slug: true,
       ...(!!includeTeamLogo ? { logo: true } : {}),
```

Comment:
@emrysal do we need to use includeTeamLogo now?

Reply:
We'll remove that eventually - for this PR we still need `logo`